Task5/BookCollection/BookFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace BookCollection
+{
```

Comment:
This is not factory because there is no internal logics, just call constructor, no need in this class